app/policies/implicit.json.erb

```diff
@@ -0,0 +1,13 @@
+{
+  "statement": [
+    {
+      "action": ["institution:read"],
+      "resource": ["institution/0"]
+    },
```

Comment:
To satisfy the spec `allows a user to read institutions even if there are no institutions on the system`